Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -43,26 +43,30 @@ Wine.prototype.DXVK = function (dxvkVersion) {
         .to(this.prefixDirectory() + "/TMP/")
         .extract();
 
+    var forEach = Array.prototype.forEach;
+    var sys32dir = this.system32directory();
     var dxvkTmpDir = this.prefixDirectory() + "/TMP/dxvk-" + dxvkVersion;
+    var self = this;
 
-    cp(dxvkTmpDir + "/x32/d3d11.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/dxgi.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/d3d10core.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/d3d10.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/d3d10_1.dll", this.system32directory());
+    forEach.call(ls(dxvkTmpDir + "/x32"), function (file) {
```

Comment:
Does `ls(dxvkTmpDir + "/x32").forEach(...)` not work?

Reply:
It works.

Reply:
I think `ls(dxvkTmpDir + "/x32").forEach(...)` is more intuitive compared to `forEach.call(ls(dxvkTmpDir + "/x32"), ...)`

Reply:
Ok, I think it does not work the other way ^^.